src/main/scala/com/globo/bigdata/gedis/JedisFactory.scala

```diff
@@ -18,6 +18,11 @@ class JedisFactory {
     new Jedis(s.host, s.port, s.connectionTimeout, s.socketTimeout)
   }
 
+  def jedis(s: ClusterSettings) : JedisCluster = {
+    import scala.collection.JavaConverters._
```

Comment:
Hi =)
why did you import `scala.collection.JavaConverters._` inside the function?

Reply:
LGTM